packages/create-bison-app/template/tests/e2e/temp/adminStorageState.json

```diff
@@ -1,49 +0,0 @@
-{
```

Comment:
These json files were probably included by accident. They are targeted in `.gitignore`.